pkg/volume/rbd/rbd_util.go

```diff
@@ -250,13 +256,13 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) (string, error) {
 
 		// Currently, we don't acquire advisory lock on image, but for backward
 		// compatibility, we need to check if the image is being used by nodes running old kubelet.
-		found, err := util.rbdStatus(&b)
+		found, rbdOutput, err := util.rbdStatus(&b)
 		if err != nil {
-			return "", err
+			return "", fmt.Errorf("error: %v, rbd output: %v", err, rbdOutput)
 		}
 		if found {
-			glog.Info("rbd is still being used ", b.Image)
-			return "", fmt.Errorf("rbd %s is still being used", b.Image)
+			glog.Info("rbd Image %v is still being used ", b.Image)
+			return "", fmt.Errorf("rbd Image %v is still being used. rbd output: %v", b.Image, rbdOutput)
```

Comment:
Use `glog.Infof` instead, and change `Image %v` to `image %s/%s` then add b.Pool in arguments.

About `<pool>/<image>` format, see comments below.